lua/acf/menu/operations/acf_menu.lua

```diff
@@ -326,4 +327,17 @@ ACF.CreateMenuOperation("Missile", "rack", "ammo crate")
 ACF.CreateMenuOperation("Engine", "engine", "fuel tank")
 ACF.CreateMenuOperation("Component", "component")
 ACF.CreateMenuOperation("Gearbox", "gearbox")
-ACF.CreateMenuOperation("Sensor", "sensor")
\ No newline at end of file
+ACF.CreateMenuOperation("Sensor", "sensor")
+
+ACF.CreateMenuOperation("1-Turret", "turret")
+ACF.CreateMenuOperation("2-Motor", "turret motor")
+ACF.CreateMenuOperation("3-Gyro", "turret gyroscope")
+ACF.CreateMenuOperation("4-Computer", "turret computer")
+
+ACF.CreateMenuOperation("Baseplate", "baseplate", nil, {
+	Text = "Convert the target entity into a baseplate. Works best on SProps rectangles (either normal, thin, or superthin). Other models not supported.",
```

Comment:
Could we just enforce a model check and send a notification when trying to convert an unsupported model instead of having a long description here?